docs/specifications/tests/Connectivity-TP/connectivity02.md

```diff
@@ -1,48 +1,193 @@
-# CONNECTIVITY02: TCP connectivity
+# CONNECTIVITY02: TCP connectivity to name servers
 
 ## Test case identifier
-
 **CONNECTIVITY02**
 
+
+## Table of contents
+
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+
 ## Objective
 
-DNS queries are sent using TCP on port 53, as described in
-[RFC 1035][RFC 1035#section-4.2.2], section 4.2.2. As specified in
-[RFC 7766][RFC 7766#section-1], section 1, support of TCP is mandatory.
+TCP is a protocol to reach a general purpose name server hosting a zone, "All
+general-purpose DNS implementations MUST support [...] TCP transport"
+([RFC 7766][RFC 7766#5], section 5).
 
-The objective for this test is that all the authoritative name servers for
-the domain are accessible over TCP on port 53.
+This Test Case will verify if the name servers in the delegation of and zone of
+*Child Zone* are reachable over TCP.
```

Comment:
This might be updated in the same manner as in https://github.com/zonemaster/zonemaster/commit/b0210f87515627540eea089a8b8254412477146e (from #1097)

Reply:
Updated.